content/posts/2019-01-03 2018-kubecon.md

```diff
@@ -16,3 +16,6 @@ It seems like there are 2 growing and diverging groups in the Kubernetes communi
 I am grateful to all of those people who worked to make the event happen. Thank you to the presenters, cooks, cleaning staff, folks who worked at information desks, drivers, vendors, volunteers, and the organizers.
 
 - Justin
+
+<!--more-->
```

Comment:
For some reason without this tag, the home page summaries render with no formatting whatsoever. If you put this tag in, they render properly. Fuck you, Hugo.